app/src/main/java/com/example/fantastic4/Star.java

```diff
@@ -0,0 +1,5 @@
+package com.example.fantastic4;
+
+public class Star extends Shape {
+    
```

Comment:
This is a test comment, add a constructor and submit a new pull request. We can then resovle this comment and merge.